app/controllers/tutors_controller.rb

```diff
@@ -98,6 +98,14 @@ def add_student
     redirect_to Tutor.of(current_user).find(tutor_id)
   end
 
+  def remove_student
+    student_id = params[:student_id]
+    tutor_id = params[:tutor_id]
+    Match.delete(student_id: student_id, tutor_id: tutor_id)
+    redirect_to Tutor.of(current_user).find(tutor_id)
+  end
```

Comment:
So there's a difference between ending a match and deleting a match. Do we want this link to actually *delete* the match so that it's gone forever, or just unmatch the student and tutor? Unmatching can be done like so:

`Match.where(student_id: student_id, tutor_id: tutor_id).update(end: Date.today)`